### DISCUSSION TITLE
windows iso editing

### DISCUSSION THREAD

**Discussion by eskehacettepe** [ventoy/Ventoy#2731] [General]
Friends, can you please explain this issue from beginning to end, I could not understand it.
https://github.com/ventoy/Ventoy/issues/647

**Comment by steve6375** [comment]
That is not a Ventoy issue 
Please use the Forum on the Ventoy website if you want a discussion.

**Reply by eskehacettepe** [reply]
There is a sources.zip file there. Could you please tell me how to run ninite.exe while formatting?
@steve6375
